packages/lockfile-lint/__tests__/main.test.js

```diff
@@ -164,6 +164,48 @@ describe('Main CLI logic', () => {
       expect(result.validatorCount).toEqual(1)
       expect(result.validatorSuccesses).toEqual(1)
     })
+
+    test('should fail with an empty yarn lock file', () => {
+      const lockfilePath = path.join(__dirname, '/fixtures/empty.json')
+      const lockfileType = 'yarn'
+      const validators = [
+        {
+          name: 'validateHosts',
+          values: ['yarn']
+        }
+      ]
+
+      expect(() =>
+        main
+          .runValidators({
+            path: lockfilePath,
+            type: lockfileType,
+            validators
+          })
+          .toThrow('Lockfile does not seem to contain a valid dependency list')
+      )
+    })
+
+    test('should fail with an empty npm lock file', () => {
+      const lockfilePath = path.join(__dirname, '/fixtures/empty.json')
+      const lockfileType = 'yarn'
+      const validators = [
+        {
+          name: 'validateHosts',
+          values: ['yarn']
```

Comment:
this should probably be 'npm', no?

Reply:
`yarn` is a valid value, so it should be fine. But I've changed it to `npm` for consistency anyways :)